packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Salaires/store/store.ts

```diff
@@ -136,6 +143,14 @@ const createSalairesStore: StoreSlice<
         ? ValidationResponse.Valid
         : ValidationResponse.NotValid;
     },
+    onPrevStep: () => {
+      matopush([
+        MatomoBaseEvent.TRACK_EVENT,
+        "outil",
```

Comment:
je pense la clé outil, on peut la remplacé par 
```suggestion
         MatomoBaseEvent.OUTIL,
```